spec/rails_helper.rb

```diff
@@ -82,6 +82,11 @@
     end
   end
 
+  # Only run specs who configure the feature flag state once
+  if ENV['DEFAULT_FEATURE_FLAG_STATE'] != 'on'
+    config.filter_run_excluding :feature_flag
+  end
+
```

Comment:
Ideally we would be able to automate excluding the specs that set the state of the feature flag so we don't test twice (and add extra time to our test build), but this is the best alternative I can think of.

Reply:
I've had a long think about the feature flags bit. So if you set all the feature flags to on and run the tests, we are activating all the feature flags and in particular tests deactivating. The opposite of that would be to still run all tests and activate/deactive the specific features, but with the other features off as there is an edge-case where if we had one feature on (the one set in specific tests) and the other off we get a failure. I think that's why it might be best to not ignore those and not run them. Let me know your thoughts